### ISSUE TITLE
Mécanique : suggestions

### ISSUE THREAD

**Issue by BellidoGilles** [redSpoutnik/Sidh-Horizons#3]
Pour mettre en avant le côté inventeur, je pensais plutôt proposé une liste de gadgets avec des restrictions au niveau des rangs. 
Quand tu prends un rang dans cette voie, tu peux prendre n'importe quel gadget de rang inférieur ou égal au tien. Possibilité d'investir plus de 5 points pour cette spécialisation.

**Comment by redSpoutnik** [comment]
Mouais, ben c'est un peu le même principe, c'est comme si pour chaque rang on mettait plein de gadgets possibles.

**Comment by BellidoGilles** [comment]
Yup, on peut aussi faire dans ce format. C'était juste au niveau de la présentation. Faut juste préciser que chaque rang peut être pris autant de fois que l'on veut.

**Comment by redSpoutnik** [comment]
Ah c'est ce que tu voulais dire? Pour moi quand tu débloque un Rang tu débloque tout ce qu'il y a dans le Rang.

**Comment by BellidoGilles** [comment]
Ah non, ça serait pété je pense. La force de l'ingenieur devrait être sa versatilité mais elle doit venir à un coût. Peut-être X pour le premier point investi dans le rang, puis 1 ou 2 pour le supplémentaire.

**Comment by redSpoutnik** [comment]
Tu as vu comment j'ai réorganisé l'arbre? Moi je préfère comme ça car les machines sont dures a construires et chères en resources, mais ça donne un nouveau *type de construction* par rang. Avec ton système faut être lvl 22 pour être très  versatile.

**Comment by BellidoGilles** [comment]
Tu parles de la branche automatisation ou de la branche mécanique. Parce que j'ai regardé vite fait et je trouve que pour la branche automatisation soit ça va être trop faible au niveau des stats si c'est max 1. Si on peut en avoir plusieurs, ça va être fumé en plus de rallonger les combats à morts. Je pensais proposer un système similaire à celui que je te proposais avec un automate de départ et tu lui ajoutes de nouvelles capacité en montant de rang. Sinon, pour la branche mécanique, avec deux ou trois gadgets par rng, t'as quand même vachement plus d'options que les autres types de spécialisation au niveau des actions. Après, la branche mécanique comme je la vois sur le github à pas vraiment cette idée de nouveaux types de construction. (Après je rappelle que je ne sais pas de quelle branche tu parles.)

**Comment by BellidoGilles** [comment]
Rang 1 : 

PV : (X,Y,Z)/2

| Fo | Ag | Res | Cha | Int | Cou |
|:-:|:-:|:-:|:-:|:-:|:-:|
| X| Y | Z | 0 | 0 | 0 |

L'ingénieur conçoit son premier robot de combat en achetant des pièces pour un total de 250 PO. L'automate prend l'apparence d'un animal d'une taille [A determiner] et aux caractéristiques X,Y,Z au choix de l'ingénieur.  La somme de X, Y et  Z ne peut pas dépasser [Rang en Automatisation + Int/2 + Ag/2]. En combat, l'automate partage la même initiative que son maître et n'agit que si ce dernier lui a donné un ordre vocal simple. 
En cas de destruction, il est possible de recréer un nouvel automate dans les mêmes conditions que ci-dessus pour un minimum de travail d'artisanat de six heures. Maximum 1 automate à tout moment.

**Comment by redSpoutnik** [comment]
Dans ce cas là on peut dire que sur une réussite critique, le robot gagne le vol.

**Comment by BellidoGilles** [comment]
Deux questions : 
- Mécaniquement, qu'est ce que ça lui apporterait de voler ? Est ce que ça peut pas attendre une montée de niveau pour se spécialiser et récuperer le vol. Est ce que ça va pas être game breaking pour toi un dispositif pour les joueurs capables de voler et potentiellement attaquer à distance.
- Pour toi il faudrait un jet pour essayer de faire l'automate. Je veux dire, le truc coûte déjà 250 PO alors si t'as une chance d'échouer en plus. C'est privé le personnage de ce qu'est censé le définir. Mais si on part sur un jet quand même, je propose temps de préparation -> difficulté du jet :
* 1h : TS[Ag,Int]
* 2h : TA[Ag,Int]
* 3h : TB[Ag,Int]
* 4h : TC[Ag,Int]
* 5h : TD[Ag,Int]
* 6h+ : Réussite automatique
ça correspond à quoi un échec ou un échec critique sur la construction ?
